README.md

```diff
@@ -36,31 +24,73 @@ Now run the following command to start using the above node version
 $ nvm use
 ```
 
-## 3. Install the dependencies
+## 2. Installation
 
-Run the following command to install the dependencies
+Run the following command to install the library
 
 ```sh
-$ npm install
+$ npm install mattermost-ui-library
 ```
 
-The following are the available scripts:
-
-## 4. Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+> The library has the following peer dependencies
```

Comment:
This is looking like this in the browser.
![image](https://user-images.githubusercontent.com/77336594/194091962-6236b961-ff62-4a39-8348-e5952b080e10.png)
Is this intentional? I think its formatting should be changed.

Reply:
Yes

Reply:
But why, what is the need for adding `>` at the beginning of the line?